includes/admin/meta-boxes/class-wc-meta-box-product-data.php

```diff
@@ -540,31 +539,23 @@ public static function output( $post ) {
 							echo esc_attr( json_encode( $json_ids ) );
 						?>" value="<?php echo implode( ',', array_keys( $json_ids ) ); ?>" /> <?php echo wc_help_tip( __( 'Cross-sells are products which you promote in the cart, based on the current product.', 'woocommerce' ) ); ?>
 					</p>
-				</div>
 
-				<div class="options_group grouping show_if_simple show_if_external">
+					<p class="form-field show_if_grouped">
+						<label for="grouped_products"><?php _e( 'Grouped products', 'woocommerce' ); ?></label>
+						<input type="hidden" class="wc-product-search" style="width: 50%;" id="grouped_products" name="grouped_products" data-placeholder="<?php esc_attr_e( 'Search for a product&hellip;', 'woocommerce' ); ?>" data-action="woocommerce_json_search_products" data-multiple="true" data-exclude="<?php echo intval( $post->ID ); ?>" data-selected="<?php
+						$product_ids = array_filter( array_map( 'absint', (array) get_post_meta( $post->ID, '_children', true ) ) );
```

Comment:
It would be better to use `WC_Product_Grouped::get_children` (and setters where we do `update_post_meta`) here.

(It seems like there are other places in the file that could be updated to use some CRUD methods as well - like upsell_ids above, but it doesn't need to be in this PR).